collector/drbd_linux.go

```diff
@@ -0,0 +1,213 @@
+// Copyright 2016 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/common/log"
+)
+
+// Numerical metric provided by /proc/drbd.
+type drbdNumericalMetric struct {
+	desc       *prometheus.Desc
+	valueType  prometheus.ValueType
+	multiplier float64
+}
+
+func newDrbdNumericalMetric(name string, desc string, valueType prometheus.ValueType, multiplier float64) drbdNumericalMetric {
```

Comment:
`newDrbdNumericalMetric` -> `newDRBDNumericalMetric` (keep case consistent within acronyms, see https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Same for other occurrences below.